samples/wfi32iot/basic-sample/src/iotconnect_app.c

```diff
@@ -379,120 +371,120 @@ bool iotconnect_sample_app(NX_IP *ip_ptr, NX_PACKET_POOL *pool_ptr, NX_DNS *dns_
 
     APP_SENSORS_init();
     
+    //Declare connection struct and initialize all values to false
+    struct click_board_connections click_board_detection;
+    click_board_detection.air7 = false;
```

Comment:
I would suggest the zero initializer like:
```c
struct click_board_connections click_board_detection = {0};
````
I am not 100% sure if some compilers will not honor it, but to be safe you can always memset(s, 0, sizeof(s)). 

This way we don't have a chance of accidentally not initializing an array member to false if we modify the struct.

This comment is optional to implenment and I will accept the reivew if it is not.